### PR TITLE
Support for different Medtronic models and fix for calculator boluses

### PR DESCRIPTION
**Pull request by gniezen**
The 523 model (used for development) has some differences compared to other pump models:
- Some records are larger
- Some units from the pump need to be divided by 40.0 instead of 10.0 (`strokesPerUnit`) to get the correct value

This PR also fixes some issues with calculator/wizard boluses and adds support for the other pump models.

### CODE REVIEW

**Comment by gniezen** [comment]
Pinging @jebeck for review.

**Comment by gniezen** [comment]
<a href="https:&#x2F;&#x2F;trello.com&#x2F;c&#x2F;4Gn33urX&#x2F;30-uploader-medtronic-calculator-data"> [uploader] Medtronic calculator data</a>

File: lib/medtronic/processData.js [hunk 1]
```diff
@@ -27,7 +27,8 @@ var savedFileEntry, fileDisplayPath;
 var debug = (typeof __DEBUG__ === 'undefined') ? false : __DEBUG__;
 var isBrowser = typeof window !== 'undefined';
 
-var larger = true; //TODO: this should be based on model number
+var larger = false; // model 523 has larger records
```
Comment (jebeck) [inline lib/medtronic/processData.js]: 523 has larger records than which other model(s)? Just for a bit more clarity here.

Comment (gniezen) [inline lib/medtronic/processData.js]: This was bothering me too. I've actually refactored the code in commit 358057cdfb9f0322c on `gniezen/medtronic-basal` to be clearer and not use global variables, i.e., it's stored in `data.settings` instead.

**Pull request merged**